test-utils/state-viewer/README.md

```diff
@@ -88,6 +88,18 @@ Flags:
 
 * `--height` takes state from the genesis up to and including the given height. By default, dumps all available state.
 
+### `dump_contracts`
+
+Dump all **distinct** contracts from the current state of the DB to JSON file.
+
+File format is a dictionary which maps account ID to base64-encoded contract code.
+
+As of 13/12/2021, takes 1m to run, size of resulting file is ~100 MB.
+
+Flags:
+
+* `--output` specifies a file to which contracts will be saved.
+
```

Comment:
Unrelated to the PR, but it seems that this info duplicates what should be available as `--help`. I'd probably default to putting most of the into into the doc-comments (which clap uses to generate help messages).